api/v1/models/proxy_status.go

```diff
@@ -54,6 +64,48 @@ func (m *ProxyStatus) Validate(formats strfmt.Registry) error {
 	return nil
 }
 
+var proxyStatusTypeEnvoyDeploymentModePropEnum []interface{}
+
+func init() {
+	var res []string
+	if err := json.Unmarshal([]byte(`["embedded","external"]`), &res); err != nil {
+		panic(err)
+	}
+	for _, v := range res {
+		proxyStatusTypeEnvoyDeploymentModePropEnum = append(proxyStatusTypeEnvoyDeploymentModePropEnum, v)
+	}
+}
+
+const (
+
```

Comment:
nit: spurious newline

Reply:
@bimmlerd 

`Code generated by go-swagger; DO NOT EDIT.`